hdfs-plugins/src/main/java/co/cask/hydrator/plugin/HDFSSink.java

```diff
@@ -51,7 +51,7 @@
 /**
  * HDFS Sink
  */
-@Plugin(type = BatchSink.PLUGIN_TYPE)
+//@Plugin(type = BatchSink.PLUGIN_TYPE)
```

Comment:
As its commented out in Jio 3.6.3 branch only, we need to make such changes more informative :
1. Pls always precede such `commented out` with a comment specifying why its being done and whether its temporary or permanent. In this case, its permanent

Reply:
https://github.com/Guavus/hydrator-plugins/commit/3767dcad1b0d74a6aef079827c50fd2a0f5b5823